modules/measurement/budget-calculator/load.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Module Name: Budget Calculator
+ * Description: Allows users to define their own metrics and generate a budget.json file from them.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Display the content of the budget calculator option page.
+ *
+ * @since 1.0.0
+ */
+function budget_calc_page_content() {
+	echo '<div class="wrap">
+	<h1>Budget Calculator</h1>
+	<form method="post" action="options.php">';
```

Comment:
If there's no additional logic here this can be split into

```suggestion
?>
	<div class="wrap">
	<h1>Budget Calculator</h1>
	<form method="post" action="options.php">
<?php
```